qiskit/visualization/qcstyle.py

```diff
@@ -57,6 +57,8 @@ def __init__(self):
             'h': 'H',
             's': 'S',
             'sdg': '$\\mathrm{S}^\\dagger$',
+            'sx': '\\sqrt{X}',
+            'sxdg': '\\sqrt{X}^\\dagger',
```

Comment:
Doesn't this need to be in mathtext mode?
```suggestion
            'sxdg': '$\\sqrt{X}^\\dagger$',
```

Reply:
absolutely, thanks!